react-common/components/extensions/ExtensionCard.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+
+interface ExtensionCardProps extends pxt.CodeCard {
+    scr: any;
+    onCardClick: (scr: any) => void;
+    loading?: boolean;
+}
+
+export const ExtensionCard = (props: ExtensionCardProps) => {
+    function handleClick() {
+        props.onCardClick(props.scr);
+    }
+
+    function handleLearnMoreClick(e: React.MouseEvent) {
+        e.stopPropagation();
+        window.open(props.learnMoreUrl, "_blank")
+    }
+
+    return <div className={(props.loading ? "loading ": "") +"ui card extensionCard"} onClick={handleClick}>
+        <img src={props.imageUrl}/>
+        <div className="content">
+            <div className="name">{props.name}</div>
+            <div className="description">{props.description}</div>
+        </div>
+
+        {props.learnMoreUrl ?
+            <div className="extra content">
+                <a className="learnmore" onClick={handleLearnMoreClick} rel="noopener noreferrer">{lf("Learn More")}</a>
+            </div>: undefined}
```

Comment:
```suggestion
            </div> : undefined}
```

nit

Reply:
maybe time to start running prettier systematically